index.ts

```diff
@@ -220,14 +226,14 @@ app.prepare().then(() => {
     }
   )
 
-  router.get('/result', async ctx => {
+  router.get('/result', async (ctx: Context) => {
     const packageString = ctx.query.p
-    ctx.redirect(`/package/${packageString.trim()}`)
+    ctx.redirect(`/package/${(packageString as string).trim()}`)
     ctx.status = 301
   })
 
-  router.get('*', async ctx => {
-    const parsedUrl = parse(ctx.req.url, true)
+  router.get('*', async (ctx: Context) => {
+    const parsedUrl = parse(ctx.req.url!, true)
```

Comment:
IMO, we should be handling null checks using a library like [ts-invariant](https://www.npmjs.com/package/ts-invariant) instead of forcing a cast to non-null